fs/littlefs/lfs_vfs.c

```diff
@@ -1019,12 +1019,14 @@ static int littlefs_bind(FAR struct inode *driver, FAR const void *data,
   fs->cfg.prog           = littlefs_write_block;
   fs->cfg.erase          = littlefs_erase_block;
   fs->cfg.sync           = littlefs_sync_block;
-  fs->cfg.read_size      = fs->geo.blocksize;
+  fs->cfg.read_size      = fs->geo.blocksize
+                           * CONFIG_FS_LITTLEFS_BLOCK_FACTOR;
```

Comment:
```suggestion
  fs->cfg.read_size      = fs->geo.blocksize *
                           CONFIG_FS_LITTLEFS_BLOCK_FACTOR;
```
I think there is no entry on the coding standard for this, but the most usual is to break the line after the operator.